10.hello-docker-compose/README.md

```diff
@@ -0,0 +1,131 @@
+# Dapr with Docker-Compose
+
+This sample demonstrates how to get Dapr running locally with Docker Compose. This uses the same applications as the `1.hello-world` sample, please review those docs for further information on the applicaiton architecture.
+
+## Prerequisites
+Clone this repo using `git clone https://github.com/dapr/samples.git` and go to the directory named */10.hello-docker-compose*
+
+- [Docker](https://docs.docker.com/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
+
+## Reviewing the Docker Compose Definition
+
+Review the Docker Compose file *./docker-compose.yml* below:
+
+```yaml
+version: '3'
+services:
+  ############################
+  # Node app + Dapr sidecar
+  ############################
+  nodeapp:
+    build: ./node
+    ports:
+      - "50002:50002"
+    depends_on:
+      - redis
+      - placement
+    networks:
+      - hello-dapr
+  nodeapp-dapr:
+    image: "daprio/daprd:edge"
+    command: ["./daprd",
+     "-app-id", "nodeapp",
+     "-app-port", "3000",
+     "-placement-address", "placement:50006",
+     "-dapr-grpc-port", "50002"]
```

Comment:
with 0.8.0 release, -components-path needs to be passed as ./components is not default anymore.